packages/generator-single-spa/src/generator-single-spa.js

```diff
@@ -28,13 +29,16 @@ module.exports = class SingleSpaGenerator extends Generator {
             { name: 'single-spa root config', value: 'root-config' },
           ]
         }
-      ])
-
-      moduleType = answers.moduleType
+      ])).moduleType
     }
 
+    console.log('moduleType', moduleType)
```

Comment:
I'm guessing we don't want to keep this here long term. Is this intentional or left over from testing?

Reply:
Removed this.